extensions/amp-bind/0.1/test/test-bind-evaluator.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {BindEvaluator, BindingDef} from '../bind-evaluator';
+
+describe('BindEvaluator', () => {
+
+  let evaluator;
+
+  beforeEach(() => {
+    evaluator = new BindEvaluator();
+  });
+
+  it('should allow callers to add bindings multiple times', () => {
+    const bindingDef = {
+      tagName: 'P',
+      property: 'text',
+      expressionString: 'oneplusone + 2',
+    };
+    const bindingDef2 = {
+      tagName: 'SPAN',
+      property: 'text',
+      expressionString: 'oneplusone + 3',
+    };
+    expect(evaluator.parsedBindings_.length).to.equal(0);
+    evaluator.addBindings([bindingDef]);
+    expect(evaluator.parsedBindings_.length).to.equal(1);
+    evaluator.addBindings([bindingDef2]);
+    expect(evaluator.parsedBindings_.length).to.equal(2);
```

Comment:
Ideally, unit tests should only test public interfaces. For this case I think it's ok, though exposing `parsedBindings_` via a new method `BindEvaluator#parsedBindingsForTesting` is more refactor-friendly than referencing the ivar directly.

I noticed that I didn't follow my own advice in `test-bind-impl.js`. 😞